### DISCUSSION TITLE
Version 1: Initial Design

### DISCUSSION THREAD

**Discussion by Phosphorus-M** [rust-lang-ar/rust-lang-ar.github.io#8] [General]
Well, this post is to discuss and show the first design of the website.
We have this:

![Desktop - 5](https://user-images.githubusercontent.com/19656993/113788987-a2b76480-9714-11eb-9f44-28600b0e074f.png)

It is a template, part of the projects and part of the members are examples and their design is attached to the changes.

**Comment by fernandosoto138** [comment]
Hi! I think it's great, Is it developed or is it only a mockup?

**Reply by LeoBorai** [reply]
A the moment is just a mockup

**Comment by igaray** [comment]
Esto luce genial, buen trabajo!

**Reply by Phosphorus-M** [reply]
Gracias! Estaremos trabajando en esto.

**Comment by PalyZambrano** [comment]
Hola Chicos!

Estuvimos revisando el diseño del sitio web con @Phosphorus-M y @EstebanBorai , basados en el trabajo que realizo @Phosphorus-M anteriormente, se ajusto la diagramación y se implementaron criterios de accesibilidad.

También se diseñaron  las paginas de _Acerca_, _Eventos_ y _Guía de Recursos_.

Algunas pantallas todavía están en estado de _mockup_, se necesita definir el contenido definitivo para pasarlas a prototipo final. Se darán cuenta de cuales están en _mockup_ porque tienen texto _Lorem Ipsum_.

También se realizó un [moodboard](https://baetica.com/que-es-moodboard/) para generar la guía de estilos.

En este link de AdobeXD podrán revisar el estado actual de el diseño:

https://xd.adobe.com/view/a6e8590a-79fc-48f7-b07d-03411c8df981-f3bb/

![image](https://user-images.githubusercontent.com/29868332/119041927-37a0b500-b985-11eb-9cc4-491569159ba9.png)

Estaria bueno tener sugerencias y feedback para mejorar el diseño actual según su criterio (:
